cloud/migrate-to-cloud/hypershift.md

```diff
@@ -0,0 +1,199 @@
+---
+title: Migrate with hypershift
+excerpt: Migrate an existing PostgreSQL database to Timescale Cloud in a single step
+product: cloud
+keywords: [data migration, hypershift]
+tags: [ingest, hypershift, postgresql]
+---
+
+# Migrate with hypershift
+
+Migrate smaller PostgreSQL databases by dumping and restoring the entire
```

Comment:
Why the focus on smaller databases? Small ones work just fine with pg_dump and pg_restore. My testing was routinely 1TB, and I don't think there's any reason it wouldn't handle 10+T. The larger you go the more the problem is with our environment and volume size limits. Testing with a multi-node target might even allow significantly larger.

Reply:
Excellent! I'll change that 👍